opencti-platform/opencti-graphql/src/database/middleware.js

```diff
@@ -307,6 +307,12 @@ const batchListThrough = async (context, user, sources, sourceSide, relationType
     targetIds = targetIds.filter((id) => elementIds.includes(id));
   }
   const targets = await elFindByIds(context, user, targetIds, opts);
+  // For each target, search if the relation is inferred
+  const findAllTargets = targets.map((target) => {
+    const findRelation = relations.find((relation) => (relation.toId === target.id) || (relation.fromId === target.id));
```

Comment:
could it happen that there are multiple relations for the same target ? since we are in a batch method, we return results for a list of entities. So we could have entityA that has relations, one inferred with targetA, and entityB that also has a relation with targetA that is not inferred, so what would happen ?